securetea/core.py

```diff
@@ -382,6 +421,7 @@ def run(self):
                 self.get_by_mice()
             else:
                 while 1:
+                    self.on_user_update()
                     # Starting mouse event listner
                     with mouse.Listener(on_move=self.on_move) as listener:
                         listener.join()
```

Comment:
Threading was kept in mind. But running on_user_update indefinitely was not a goal for me. On detection of any kind of malicious activity it should be run was my approach. There is no need to keep on running it. This would save time and computational power important for IoT's

Reply:
@kUSHAL0601 If we are not running in a monitoring loop, then how are we monitoring the addition of new connections?

Reply:
Yeah will run threads and update it.